### PR TITLE
Add f as optional

### PR DESCRIPTION
**Pull request by gmuraru**
## Types of changes

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: crypten/__init__.py [hunk 1]
```diff
@@ -236,7 +236,9 @@ def load(f, preloaded=None, encrypted=False, dummy_model=None, src=0, **kwargs):
 
         # source party
         if comm.get().get_rank() == src:
-            result = preloaded if preloaded else torch.load(f, **kwargs)
+            assert not (f is None and preloaded is None), "Load failed: f or preloaded should be specified"
```
Comment (gmuraru) [inline crypten/__init__.py:239]: If someone adds ```preloaded``` then we should not pass a loading function

Comment (knottb) [reply crypten/__init__.py:239]: This seems reasonable. 

We're currently working on refactoring the `load` function to deprecate `preloaded` and separate it into a new function.

Comment (gmuraru) [reply crypten/__init__.py:239]: In this case, should those changes be merged or should we wait for the refactoring?

**Comment by gmuraru** [comment]
@knottb do you have an estimated time when the separation will be ready? (until then could we merge those changes or it would not make sense?)

**Comment by gmuraru** [comment]
Will close this as it was fixed [here](https://github.com/facebookresearch/CrypTen/commit/1b295d9422636de37a7e817d349f86dfd973293b)

**Pull request closed without merging**